crates/samples/echo2/src/main.rs

```diff
@@ -32,6 +38,7 @@ fn main() -> windows::core::Result<()> {
     Ok(())
 }
 
+// validates the configs in the config package have the right values.
 fn validate_configs(actctx: &ActivationContext) {
```

Comment:
Will this fail if there is no confi? seems lke is the case

Reply:
The app will crash if config is not found or fail to load. This is expected behavior and we rely this in CI to make sure the config is actually working. If the app crashes , the ci should fail.